pkg/virt-controller/watch/vm_test.go

```diff
@@ -1384,7 +1522,7 @@ var _ = Describe("VirtualMachine", func() {
 				Expect(vm.Status.StartFailure.ConsecutiveFailCount).To(Equal(2))
 			})
 
-			It("should clear start failures when VMI hits running state", func() {
+			PIt("should clear start failures when VMI hits running state", func() {
```

Comment:
Might be worth creating an issue about this? Or at least a comment that explains the problem?